plutus-tx-plugin/test/Array/9.6/compiledIndexArray.uplc.golden

```diff
@@ -0,0 +1 @@
+(program 1.1.0 (force indexArray [I 1, I 2, I 3] 2))
```

Comment:
Why do the elements inside the array have `I` constructors, while the integer argument to `indexArray` (`2`) doesn't?

Reply:
Idk, looks like a pretty-printing issue to me